monit/files/modules

```diff
@@ -1,50 +1,17 @@
-# -*- coding: utf-8 -*-
-# vim: ft=jinja
```

Comment:
Is there a reason for removing these?

Reply:
No, my version is behind that one anyway. I've added these lines to my file.